spec-draft.md

```diff
@@ -136,6 +136,106 @@ Example payload for setting "Airplane Mode":
 
+Device Orientation
+------------------
+
+The [JSON Wire Protocol](https://code.google.com/p/selenium/wiki/JsonWireProtocol#/session/:sessionId/orientation)
+already supports a very basic concept of
+orientation, involving two orientation options: portrait and landscape.
+Ultimately this will be subsumed under Device Rotation (see below), but for now
+we continue to support these endopints for mobile devices.
+
+* GET /session/:sessionid/orientation
+    * returns ScreenOrientation
+* POST /session/:sessionid/orientation
+    * accepts a ScreenOrientation
+
+### ScreenOrientation
+
+A `ScreenOrientation` can be one of two strings:
+
+* `LANDSCAPE`
+* `PORTRAIT`
+
+Device Rotation
+---------------
+
+Mobile devices can theoretically be used in any spatial orientation. Thus the
+concept of 'orientation' is actually much more general. We will call it
+'rotation'. Currently, mobile OS automation technologies do not expose hooks
+for arbitrary rotation, but allow simply for a small set of 'orientations',
+like landscape or portrait (largely because these are the main categories used
+by apps themselves). We described some simple 'orientation' options above. Here
+we describe the spec for rotation, which can be used when automation tool
+vendors release support for more than 'landscape' and 'portrait' orientations.
+Apple, for example, already has more options (such as "portrait + faceup").
+
+We can define any rotation with a combination of 3 angles: rotations around the
+X, Y, and Z axes in 3d space. The X axis is horizontal (width), Y is vertical
+(height), and Z is extensive (depth). We arbitrarily stipulate the "default"
+position for a device (i.e., its position when the X, Y and Z angles are 0, 0,
+and 0) is as follows:
+
+* The device screen is facing us, parallel to the value 0 on the Z axis (i.e.,
+  it extends into the positive Z space, not the negative (see below))
+* The bottom edge of the device lies on the X axis The left edge of the device
+  lies on the Y axis
+
+In terms of polarity, we stipulate that the X axis goes from negative to
+positive, left to right. The Y axis goes from negative to positive, bottom to
+top. And the Z axis goes from negative to positive, front to back.
+
+Now, assuming the bottom, left, front edge of the device always remains at the
+origin point (i.e., the point x=0, y=0, z=0), we can define any rotation. The
+"default" rotation described above would be equivalent to a normal "portrait"
+orientation, the device being held straight out in front of the user. The
+rotation `(x=90deg, y=0deg, z=0deg)` signifies a similar portrait orientation,
+but "face up" on a table, for example. "Face down portrait" would then be
+`(x=90deg, y=0deg, z=180deg)`.
+
+We can now define the necessary endpoints:
+
+* GET /session/:sessionid/rotation
+    * returns DeviceRotation
+* POST /session/:sessionid/rotation
+    * accepts a DeviceRotation
+
+The remote end MUST reply with the capability "deviceRotationEnabled" in order
```

Comment:
the capabilitiy should be `deviceRotation` to match the rest of the [WebDriver Capabilities](http://w3c.github.io/webdriver/webdriver-spec.html#capabilities)